doc/usage2rst.sh

```diff
@@ -8,6 +8,8 @@ if [ $1 == "--help" ]; then
 fi
 
 real=$(realpath $1)
+# githash=`git rev-parse HEAD`
+githash=8574aa8e0d5968916df1df42c28621a48b2a5281
```

Comment:
what's this?
is it OK to fix the hash?

Reply:
@sw005320 
Sorry, this is a debug code! I will check all diffs and fix debug lines!!